test/SPIRV/fmod.ll

```diff
@@ -1,51 +0,0 @@
-; __kernel void fmod_kernel( float out, float in1, float in2 )
```

Comment:
I think we need to keep this test but modify it to match expected result. I think it should be translated to extended instruction OpenCL.fmod, right?